set.mm

```diff
@@ -530624,6 +530625,132 @@ property of surreals and will be used (via surreal cuts) to prove many
+      ( vz vw vx cv cbday cfv wss wcel wi csur wral wa wceq csslt wbr syl cslt
+      cmade cleft cright cscut co csn crab cima ssltleft adantl ssltright fveq2
+      eqimss a1i wne ssltsep vex breq2 ralsn ralbii sylib breq1 ralbidv anim12i
+      cint weq cold wb leftval raleqdv rightval anbi12d ralrab anbi12i ad2antlr
+      syl6bb simplrl sltirr con0 bdayelon ontri1 con2bii simplll madebdaylemold
+      wn mp2an mp3an2i wo slttrine ad2ant2lr simprrr imbi12d com23 simprrl jaod
+      rspccv sylbid imp syld syl5bir mt3d expr impr sylanr2 pm2.61dne ralrimiva
+      ex wfn bdayfn ssrab2 fnssintima sneq breq2d breq1d bitri sylibr simpr jca
+      elrabd fnfvima mp3an12i intss1 eqssd w3a lltropt eqscut mpancom mpbir3and
+      ) AGZHICGZJYIYJUAIKLAMNCBHIZNZBMKZOZBUBIZBUCIZUDUEBPZYOBUFZQRZYRYPQRZYKHY
+      ODGZUFZQRZUUBYPQRZOZDMUGZUHZVEZPZYMYSYLBUIUJZYMYTYLBUKUJZYNYKUUHYNYOEGZUF
+      ZQRZUUMYPQRZOZYKUULHIZJZLZEMNZYKUUHJZYNUUSEMYNUULMKZUUPUURYNUVBUUPOOZUURB
+      UULBUULPZUURLUVCUVDYKUUQPUURBUULHULYKUUQUMSUNUUPYNUVBFGZUULTRZFYONZUULUVE
+      TRZFYPNZOZBUULUOZUURLZUUNUVGUUOUVIUUNUVEYITRZAUUMNZFYONUVGFAYOUUMUPUVNUVF
+      FYOUVMUVFAUULEUQZYIUULUVETURUSUTVAUUOYIUVETRZFYPNZAUUMNUVIAFUUMYPUPUVQUVI
+      AUULUVOAEVFUVPUVHFYPYIUULUVETVBVCUSVAVDYNUVBUVJUVLYNUVBOUVJUVEBTRZUVFLZFY
+      KVGIZNZBUVETRZUVHLZFUVTNZOZUVLYMUVJUWEVHYLUVBYMUVJUVFFUUABTRZDUVTUGZNZUVH
+      FBUUATRZDUVTUGZNZOUWEYMUVGUWHUVIUWKYMUVFFYOUWGDBVIVJYMUVHFYPUWJDBVKVJVLUW
+      HUWAUWKUWDUWFUVRUVFFDUVTUUAUVEBTVBVMUWIUWBUVHFDUVTUUAUVEBTURVMVNVPVOYNUVB
+      UWEUVLYNUVBUWEOZOZUVKUURUWMUVKOZUURUULUULTRZUWNUVBUWOWEYNUVBUWEUVKVQZUULV
+      RSUURWEUUQYKKZUWNUWOUURUWQYKVSKZUUQVSKUURUWQWEVHBVTZUULVTYKUUQWAWFWBUWNUW
+      QUULUVTKZUWOUWRUWNYLUVBUWQUWTLUWSYLYMUWLUVKWCUWPAYKUULCWDWGUWMUVKUWTUWOLZ
+      UWMUVKBUULTRZUULBTRZWHZUXAYMUVBUVKUXDVHYLUWEBUULWIWJUWMUXBUXAUXCUWMUWTUXB
+      UWOUWMUWDUWTUXBUWOLZLYNUVBUWAUWDWKUWCUXEFUULUVTFEVFZUWBUXBUVHUWOUVEUULBTU
+      RUVEUULUULTURWLWPSWMUWMUWTUXCUWOUWMUWAUWTUXCUWOLZLYNUVBUWAUWDWNUVSUXGFUUL
+      UVTUXFUVRUXCUVFUWOUVEUULBTVBUVEUULUULTVBWLWPSWMWOWQWRWSWTXAXGXBWQXCXDXEXB
+      XFUVAUUREUUFNZUUTHMXHZUUFMJZUVAUXHVHXIUUEDMXJZEMUUFYKHXKWFUUEUUPUUREDMDEV
+      FZUUCUUNUUDUUOUXLUUBUUMYOQUUAUULXLZXMUXLUUBUUMYPQUXMXNVLVMXOXPYNYKUUGKZUU
+      HYKJUXIUXJYNBUUFKUXNXIUXKYNUUEYSYTODBMUUABPZUUCYSUUDYTUXOUUBYRYOQUUABXLZX
+      MUXOUUBYRYPQUXPXNVLYLYMXQYNYSYTUUJUUKXRXSMUUFHBXTYAYKUUGYBSYCYMYQYSYTUUIY
+      DVHZYLYOYPQRYMUXQBYEDYPYOBYFYGUJYH $.
+  $}
+
+  ${
+    $d A a $.  $d a b $.  $d a x $.  $d A x $.  $d a y $.  $d b w $.  $d b x $.
+    $d b y $.  $d b z $.  $d l r $.  $d l x $.  $d r x $.  $d w x $.  $d w y $.
+    $d w z $.  $d X x $.  $d x y $.  $d x z $.  $d y z $.
+    $( A surreal is part of a made set iff its birthday is less than or equal
+       to that ordinal.  (Contributed by Scott Fenton, 19-Aug-2024.) $)
+    madebday $p |- ( ( A e. On /\ X e. No ) -> ( X e. ( _M ` A ) <->
+    ( bday ` X ) C_ A ) ) $=
```

Comment:
Maybe it's because I'm not versed into surreal numbers, but at first I could not guess which _that_ ordinal was, since no ordinal was mentioned before. Would it make sense to write:
> A surreal is part of the set made by ordinal `A` iff its birthday is less than or equal to `A`

?

Reply:
Good point. That's done now.